src/compoz.ts

```diff
@@ -336,6 +341,26 @@ export class Compoz {
 				this.cfg.onContentChange(contentHTML)
 			}
 		}
+
+		this.elInput.onkeydown = e => {
+			if (this.currentHeight !== this.elInput.offsetHeight) {
+				if (
+					e.which === this.keyCodeEnter ||
+					e.which === this.keyCodeDelete ||
+					e.which === this.keyCodeBackspace
+				) {
+					if (
+						this.elInput.offsetHeight < this.defaultInputMaxHeight
+					) {
+						this.cfg.onChangeHeight()
+						this.currentHeight = this.elInput.offsetHeight
+					}
+				}
+			} else {
+				return
```

Comment:
No need for "else" condition, or return first on "this.currentHeight === this.elInput.offsetHeight".